src/manifest/parsers/macroParser.ts

```diff
@@ -61,11 +61,31 @@ export class MacroParser {
                 new RegExp(`test\\s${name.replace("test_", "")}\\(`),
               )
             ) {
+              let description = "";
+              let args = [];
+      
+              // Parse macro description
+              for (let i = index - 1; i >= 0; i--) {
+                const line = macroFileLines[i].trim();
+                if (line.startsWith("/*") || line.startsWith('"""')) {
+                  description = line.slice(2, -2).trim();
+                  break;
+                }
+              }
+      
+              // Parse macro arguments
+              const argsMatch = currentLine.match(/\((.*)\)/);
+              if (argsMatch) {
+                args = argsMatch[1].split(",").map(arg => arg.trim());
+              }
+      
```

Comment:
instead of this change, you could get description and arguments from `macro` in line no 37